src/emissions/APU_emissions.py

```diff
@@ -55,6 +57,11 @@ def get_APU_emissions(
         APU_PM10 - APU_emission_indices['PMnvol']
     ).item()
 
+    if nvpm_method == "SCOPE11":
```

Comment:
I would add a comment on all the lines where you're setting these values to zero to say why (because they're things that weren't supported in "old AEIC" but you want to support in the new code, or whatever).